major_event_log/models.py

```diff
@@ -30,6 +30,12 @@ class Event(models.Model):
                                     "Name will appear as the Reporting Agent")
     contact_email = models.EmailField()
 
+    def is_success(self):
+        if self.get_outcome_display() == "Success":
```

Comment:
I like it. I did try comparing the value of outcome to the uri, but that seemed a bit messy. This will clean that up nicely, and remove comparisons with hardcoded strings